mz_bokeh_package/utilities/current_user.py

```diff
@@ -0,0 +1,87 @@
+import os
+from bokeh.io import curdoc
+
+from mz_bokeh_package.utilities.environment import Environment
+from mz_bokeh_package.utilities.graphql_api import MZGraphQLClient
+
+
+class CurrentUser:
+    """
+    Class with static methods for getting information about the current user
+    """
+
+    _users = {}
+
+    @classmethod
+    def get_user_info(cls, api_key: str | None) -> dict:
+        """Retrieves user information for the current session or API key, caching the information for future use.
+
+        Args:
+            api_key: unique user api key.
+
+        Returns:
+            dictionary of user "id" and "name".
```

Comment:
Since this is a package that is imported by other repos, we have to make sure that the documentation is as precise as possible. Thus, here, I would make it more explicit, like:

```
Returns:
   a dictionary containing the user id and user name corresponding to the user with this api_key:
      {"id": <user id>, "name": <user name>}
```